pkgs/applications/misc/with-shell/default.nix

```diff
@@ -0,0 +1,21 @@
+{ stdenv, fetchFromGitHub }:
+stdenv.mkDerivation {
+  name = "with";
```

Comment:
Add version to `name`.
```nix
name = "with-0.0.1";
```

`version` field is usually used as a variable to reuse later and is not used by `mkDerivation` itself, so you may remove it.